core/tests/test_utils.py

```diff
@@ -396,9 +514,11 @@ def set_collection_editors(self, collection_editor_usernames):
             self.set_user_role(name, feconf.ROLE_ID_COLLECTION_EDITOR)
 
     def get_current_logged_in_user_id(self):
+        """Returns the login user id."""
         return os.environ['USER_ID']
 
     def get_user_id_from_email(self, email):
+        """Returns the email id of the login user."""
```

Comment:
This is incorrect. It should be: `Returns the user ID of the user with the given email.`

Also, the args section is missing.